Classes/Domain/Repository/AjaxApplicationRepository.php

```diff
@@ -25,8 +25,22 @@ public function getTotalResultsOfQuery(ApplicationQuery $query)
         /** @var QueryBuilder */
         $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)->getQueryBuilderForTable('tx_ats_domain_model_application');
 
-        $count = $queryBuilder->count('uid')
-            ->from('tx_ats_domain_model_application')
+        $queryBuilder->getRestrictions()
+            ->removeByType(StartTimeRestriction::class)
+            ->removeByType(EndTimeRestriction::class)
+            ->removeByType(HiddenRestriction::class);
```

Comment:
Why are these removed?

Reply:
Because of the join to the jobs table. With applications, these fields are not used anyway, but jobs may be hidden or limited for frontend access via start/endtime. Since we are in backend context, these jobs (and connected applications) still need to show up.

Reply:
Still sounds like this should be done separately then.

Reply:
Where exactly should it be done?

Reply:
In a separate PR/commit AFAIS.

Reply:
As discussed I get the intention now, so please add a small comment here and below to quickly explain this. Otherwise the next one who looks at this will have the same question. Possibly me. ;-)